packages/fxa-settings/src/components/Settings/index.test.tsx

```diff
@@ -30,6 +30,15 @@ jest.mock('../../lib/totp-utils', () => {
   };
 });
 
+// Mocking the component here avoids dealing with all the dependecies required to start the 2fa setup flow
```

Comment:
@nshirley I would appreciate your opinion on the changes in this files, especially mocking the component here + suppressing an expected console warning from test output :)